spec/docs/v3/participant_declarations_spec.rb

```diff
@@ -15,174 +15,151 @@
-                        created_at: "2020-11-13T11:21:55Z",
-                        delivery_partner_id: nil,
-                        statement_id: "1cceffd7-0efd-432a-aedc-7be2d6cc72a2",
-                        clawback_statement_id: nil,
-                        ineligible_for_funding_reason: nil,
-                        mentor_id: nil,
-                        uplift_paid: true,
-                        evidence_held: nil,
-                      },
+                  style: :deepObject,
+                  explode: true,
+                  required: false,
+                  description: "Refine participant declarations to return.",
+                  example: CGI.unescape({
+                    participant_id: "ab3a7848-1208-7679-942a-b4a70eed400a",
+                    updated_since: "2020-11-13T11:21:55Z",
+                  }.to_param)
+
+        parameter name: :page,
+                  in: :query,
+                  schema: {
+                    "$ref": "#/components/schemas/Pagination",
+                  },
+                  style: :deepObject,
+                  explode: true,
+                  required: false,
+                  example: CGI.unescape({ page: { page: 1, per_page: 5 } }.to_param),
+                  description: "Pagination options to navigate through the list of participant declarations."
+
+        response "200", "A list of participant declarations" do
+          schema({ "$ref": "#/components/schemas/MultipleParticipantDeclarationsResponse" })
+
+          after do |example|
+            content = example.metadata[:response][:content] || {}
+            example_spec = {
+              "application/json" => {
+                examples: {
+                  success: {
+                    value: {
+                      data: [
+                        {
+                          id: "db3a7848-7308-4879-942a-c4a70ced400a",
+                          type: "participant-declaration",
+                          attributes: {
+                            participant_id: "08d78829-f864-417f-8a30-cb7655714e28",
+                            declaration_type: "started",
+                            declaration_date: "2020-11-13T11:21:55Z",
+                            course_identifier: "ecf-induction",
+                            state: "eligible",
+                            updated_at: "2020-11-13T11:21:55Z",
+                            created_at: "2020-11-13T11:21:55Z",
+                            delivery_partner_id: "99ca2223-8c1f-4ac8-985d-a0672e97694e",
+                            statement_id: "99ca2223-8c1f-4ac8-985d-a0672e97694e",
+                            clawback_statement_id: nil,
+                            ineligible_for_funding_reason: nil,
+                            mentor_id: "907f61ed-5770-4d38-b22c-1a4265939378",
+                            uplift_paid: true,
+                            evidence_held: "other",
+                            has_passed: true,
```

Comment:
```suggestion
                            has_passed: nil,
```

Reply:
could we do the same in all examples everywhere else unless it's an NPQ and completed declaration? Also could you amend the `v3/api_spec` as it includes those examples please?